docs/feature_mouse_keys.md

```diff
@@ -55,27 +62,55 @@ The default speed for controlling the mouse with the keyboard is intentionally s
 #define MOUSEKEY_WHEEL_TIME_TO_MAX 40
 ```
 
-
-### `MOUSEKEY_DELAY`
+#### `MOUSEKEY_DELAY`
 
 When one of the mouse movement buttons is pressed this setting is used to define the delay between that button press and the mouse cursor moving. Some people find that small movements are impossible if this setting is too low, while settings that are too high feel sluggish.
 
-### `MOUSEKEY_INTERVAL`
+#### `MOUSEKEY_INTERVAL`
 
 When a movement key is held down this specifies how long to wait between each movement report. Lower settings will translate into an effectively higher mouse speed.
 
-### `MOUSEKEY_MAX_SPEED`
+#### `MOUSEKEY_MAX_SPEED`
 
 As a movement key is held down the speed of the mouse cursor will increase until it reaches `MOUSEKEY_MAX_SPEED`.
 
-### `MOUSEKEY_TIME_TO_MAX`
+#### `MOUSEKEY_TIME_TO_MAX`
 
 How long you want to hold down a movement key for until `MOUSEKEY_MAX_SPEED` is reached. This controls how quickly your cursor will accelerate.
 
-### `MOUSEKEY_WHEEL_MAX_SPEED`
+#### `MOUSEKEY_WHEEL_MAX_SPEED`
 
 The top speed for scrolling movements.
 
-### `MOUSEKEY_WHEEL_TIME_TO_MAX`
+#### `MOUSEKEY_WHEEL_TIME_TO_MAX`
 
 How long you want to hold down a scroll key for until `MOUSEKEY_WHEEL_MAX_SPEED` is reached. This controls how quickly your scrolling will accelerate.
+
+### Using Constant Cursor Speeds
+
+You can define three different speeds (slow, medium, fast) for both the cursor and the mouse wheel, and also whether speed selection is momentary or tap-to-select.
+
+For each speed you can specify the interval between reports in milliseconds and how many pixels you want the cursor to move per report (offset).
+
+Add the following settings to your keymap's `config.h` file and adjust them to your liking:
+
+```
+#define MK_3_SPEED              // enable constant speeds
+#define MK_MOMENTARY_ACCEL      // comment this out for tap-to-select acceleration
+// cursor speeds:
+#define MK_C_OFFSET_SLOW     1  // pixels
+#define MK_C_INTERVAL_SLOW 100  // milliseconds
+#define MK_C_OFFSET_MED      4
+#define MK_C_INTERVAL_MED   16
+#define MK_C_OFFSET_FAST    12
+#define MK_C_INTERVAL_FAST  16
+// scroll wheel speeds:
+#define MK_W_OFFSET_SLOW     1  // wheel clicks
+#define MK_W_INTERVAL_SLOW 400
+#define MK_W_OFFSET_MED      1
+#define MK_W_INTERVAL_MED  200
+#define MK_W_OFFSET_FAST     1
+#define MK_W_INTERVAL_FAST 100
```

Comment:
As above, could this be reformatted into a table to follow the format the rest of the docs are moving to?